services/bundle_analysis/notify/__init__.py

```diff
@@ -35,9 +35,9 @@ class NotificationFullContext(NamedTuple):
 
 
 class BundleAnalysisNotifyReturn(NamedTuple):
-    notifications_configured: tuple[NotificationType]
-    notifications_attempted: tuple[NotificationType]
-    notifications_successful: tuple[NotificationType]
+    notifications_configured: tuple[NotificationType, ...]
+    notifications_attempted: tuple[NotificationType, ...] | tuple[Never]
+    notifications_successful: tuple[NotificationType, ...] | tuple[Never]
```

Comment:
Is there a reason to specify `tuple[Never]`? I believe `tuple[NotificationType, ...]` is inclusive of empty tuples as well.